app/src/renderer/components/common/NiDataEmpty.vue

```diff
@@ -0,0 +1,15 @@
+<template lang="pug">
+data-msg(icon="info_outline")
+  div(slot="title") N/A
```

Comment:
if i understand this correctly, this component will be used when there are no errors but also no data to display. this is a great first step but in this situation we could provide some context + a call to action. 

for example, on the transaction history page it could say ...

```
(slot="title") No Transaction History 
(slot="subtitle") Looks like you haven't sent or received any transactions yet. Head over to the [send page]() to make your first transaction!
```

Reply:
This is a great idea. I'll push an update to this effect:

<img width="1280" alt="screen shot 2017-11-28 at 12 00 17 pm" src="https://user-images.githubusercontent.com/172531/33300486-efab236e-d433-11e7-802e-8f532da1bda0.png">